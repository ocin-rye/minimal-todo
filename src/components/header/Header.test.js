import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import Header from './index';

describe('<Header />', () => {
  it('renders without crashing', () => {
    const header = shallow(<Header />);
    expect(toJson(header)).toMatchSnapshot();
  });
});
