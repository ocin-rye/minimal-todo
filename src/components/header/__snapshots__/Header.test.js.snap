// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders without crashing 1`] = `
<div
  className="container"
>
  <h1
    className="title"
  >
    Minimal Todo App
  </h1>
</div>
`;
